external_api/lib/dictionary_utils.py

```diff
@@ -0,0 +1,100 @@
+from external_api.exceptions import APIHandlerException
```

Comment:
W1618 import missing `from __future__ import absolute_import`

Reply:
For this file add a note that `word` and `value` are one and the same thing